tools/src/react-native-nightlies/patches/lottie-react-native.patch

```diff
@@ -0,0 +1,32 @@
+--- a/node_modules/lottie-react-native/android/src/main/java/com/airbnb/android/react/lottie/LottieAnimationViewPropertyManager.kt
++++ b/node_modules/lottie-react-native/android/src/main/java/com/airbnb/android/react/lottie/LottieAnimationViewPropertyManager.kt
+@@ -18,7 +18,6 @@ import com.facebook.react.bridge.ReadableArray
+ import com.facebook.react.bridge.ReadableMap
+ import com.facebook.react.bridge.ReadableType
+ import com.facebook.react.views.text.ReactFontManager
+-import com.facebook.react.views.text.TextAttributeProps.UNSET
+ import com.facebook.react.util.RNLog
+ import java.lang.ref.WeakReference
+ import java.util.regex.Pattern
+@@ -69,7 +68,7 @@ class LottieAnimationViewPropertyManager(view: LottieAnimationView) {
+         view.setFontAssetDelegate(object : FontAssetDelegate() {
+             override fun fetchFont(fontFamily: String): Typeface {
+                 return ReactFontManager.getInstance()
+-                    .getTypeface(fontFamily, UNSET, UNSET, view.context.assets)
++                    .getTypeface(fontFamily, -1, -1, view.context.assets)
+             }
+         
+             override fun fetchFont(fontFamily: String, fontStyle: String, fontName: String): Typeface {
+@@ -80,10 +79,10 @@ class LottieAnimationViewPropertyManager(view: LottieAnimationView) {
+                     "Medium" -> 500
+                     "Bold" -> 700
+                     "Black" -> 900
+-                    else -> UNSET
++                    else -> -1
```

Comment:
Is it safe to use -1 here? What something changes in the future? Are they going to make it public again?

Reply:
@matinzd it looks not safe, i don't have much insight of how text processing between react-native and lottie-react-native. it's just an intermediate patch for us to fix breaking changes before 3rd party libs publish new versions.